liri.js

```diff
@@ -67,6 +67,70 @@ function myTweets() {
 }
```

Comment:
console.log's are an expensive process and should not be called more than it needs to be, instead I would just log the whole body object.